tests/test_runlib.py

```diff
@@ -554,6 +585,40 @@ def test_in_toto_wrong_key(self):
       in_toto_run(self.step_name, None, None,
           ["python", "--version"], True, self.key_pub)
 
+  def test_nonexistent_directory(self):
+    """Fail run, passed metadata_directory not exist. """
+    expected_error = IOError if sys.version_info < (3, 0) \
+        else FileNotFoundError
+    with self.assertRaises(expected_error):
+      in_toto_run(self.step_name, None, None, ["python", "--version"],
+          True, self.key, metadata_directory='nonexistentDir')
+
+  def test_not_a_directory(self):
+    """Fail run, passed metadata_directory is not a directory. """
+    fd, path = tempfile.mkstemp()
+    os.write(fd, b"hello in-toto")
+    os.close(fd)
+    # Windows will raise FileNotFoundError instead of NotADirectoryError
+    expected_error = IOError if sys.version_info < (3, 0) \
+       else (NotADirectoryError, FileNotFoundError)
```

Comment:
```suggestion
        else (NotADirectoryError, FileNotFoundError)
```
Same indentation mistake exists in some of the other ternary expressions below. Please change there too.